lib/Wizard.php

```diff
@@ -685,6 +685,17 @@ public function guessBaseDN() {
 			}
 		}
 
+		//check whether the agent name is in email form
+		$i = \stripos($this->configuration->ldapAgentName, '@');
+		if ($i !== false) {
+			$base1 = \substr($this->configuration->ldapAgentName, $i+1);
+			$base1 = 'dc=' . \implode(',dc=', \explode('.', $base1));
+			if ($this->testBaseDN($base)) {
+				$this->applyFind('ldap_base', $base);
```

Comment:
I guess this should be `$base1` instead of `$base`

Reply:
oops, you are right, of course.
Thanks. pushed a fix.